api.jsx

```diff
@@ -16,4 +16,14 @@ async function getArticles() {
     }
 }
 
-export default getArticles
\ No newline at end of file
+// Functions. 
+export async function getArticleById(articleId) {
+    try {
+        const response = await ncNewsApi.get(`/articles/${articleId}`)
+
+        return response.data.article
+    }
+    catch (err) {
+        console.log(err)
```

Comment:
Could you use this to set error state to provide user feedback if an error did occur?